src/components/Footer.tsx

```diff
@@ -85,6 +86,19 @@ const FooterGrid = styled.div`
     grid-template-columns: 1fr;
   }
 `;
+
+const SocialMediaIcons = styled.div`
+  a {
+    display: inline-block;
+    color: black;
+    margin-right: 1rem;
+  }
+  svg {
+    height: 1.4rem;
+    width: 1.4rem;
```

Comment:
For me those icons seem small (on desktop), what do you think? Maybe we should make a little bit bigger?

Reply:
Yeah, changed to `1.7rem`.